master/internal/experiment.go

```diff
@@ -345,7 +345,6 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 			e.Config.CheckpointStorage().SaveExperimentBest(),
 			e.Config.CheckpointStorage().SaveTrialBest(),
 			e.Config.CheckpointStorage().SaveTrialLatest(),
-			true,
 		)
 		if err != nil {
 			ctx.Log().WithError(err).Error("")
```

Comment:
we should use `newCheckpointGCTask` below, too (we may need to change the singature a tad).